data/near earth jobs.txt

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Historical field trip to <planet>"
+	job
+	repeat
+	description `These <bunks> school students need to reach <destination> by <date> for a historical field trip. The school will pay you <payment>.`
+	passengers 20 100
+	deadline
+	to offer
+		random < 10
+	source
+		attributes "deep" "paradise"
+	destination
+		distance 4 12
+		attributes "near earth"
+	on complete
+		payment
+		payment 10000
+		dialog `The students excitedly rush off your ship, taking pictures of everything they see. The school deposits <payment> into your account.`
+
+
+mission "To Earth Day celebration [0]"
+	name `Earth Day celebration`
+	job
+	repeat
+	description `Bring <fare> to <destination> by <date> for the Earth Day celebration. Payment is <payment>.`
+	passengers 4 4 .5
+	deadline
+	to offer
+		random < 80
+		or
+			and
+				month == 2
```

Comment:
You could put additional `random` checks in these `and` nodes to roughly gate the offer chance.. e.g. 20% offer in late feb, 40% in March, and the full 80% in April.